### PR TITLE
[Docs] SSL-config related changes in the SequelizeORM docs

### PR DESCRIPTION
**Pull request by priyanshi-yb**
This PR includes the changes to mention the steps of how to connect the Sequelize ORM app with YugabyteDB Cloud Cluster.

### CODE REVIEW

File: docs/content/preview/quick-start/build-apps/nodejs/ysql-sequelize.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ showAsideToc: true
 
 ## Prerequisites
 
-This tutorial assumes that you have installed YugabyteDB and created a cluster. Refer to [Quick Start](../../../../quick-start/).
+This tutorial assumes that you have installed YugabyteDB and created a cluster. Refer to [Quick Start](../../../../quick-start/). Also, you have installed latest stable version of [node](https://nodejs.org/en/).
```
Comment (ashetkar) [inline docs/content/preview/quick-start/build-apps/nodejs/ysql-sequelize.md:39]: Please make these two as separate points.

File: docs/content/preview/drivers-orms/nodejs/sequelize-yugabytedb.md [hunk 2]
```diff
@@ -180,6 +180,31 @@ Employees Details:
 ]
 ```
 
+## Using Cloud Cluster to connect with Seqeulize:
```
Comment (ashetkar) [inline docs/content/preview/drivers-orms/nodejs/sequelize-yugabytedb.md:183]: Can this be something like "Specifying SSL configuration"?
And then we can mention that this is required if one is connecting to a YB Managed cluster or even a local YB cluster with SSL enabled.

File: docs/content/preview/drivers-orms/nodejs/sequelize-yugabytedb.md [hunk 3]
```diff
@@ -179,6 +179,32 @@ Employees Details:
...
+    port: 5433,
+    host: "<host_name>",
+    dialectOptions: {
+        ssl: {
+            rejectUnauthorised: true,
+            ca: fs.readFileSync('<path_to_root_crt>').toString(),
+        }
+    }
+  });
+```
```
Comment (polarweasel) [inline docs/content/preview/drivers-orms/nodejs/sequelize-yugabytedb.md:207]: Gotta use markdownlint... Also, this should be a numbered list.

````suggestion

## Specifying SSL configuration

This configuration can be used while connecting to a YB Managed cluster or a local YB cluster with SSL enabled.

1. Install the `fs` package to read the SSL certificate:

    ```sh
    npm install fs
    ```

1. Add the following line to use the `fs` module:

    ```js
    const fs = require('fs');
    ```

1. Use the following configuration in the `models/index.js` file when you create the sequelize object:

    ```js
    const sequelize = new Sequelize("<db_name>", "<user_name>","<password>" , {
        dialect: 'postgres',
        port: 5433,
        host: "<host_name>",
        dialectOptions: {
            ssl: {
                rejectUnauthorised: true,
                ca: fs.readFileSync('<path_to_root_crt>').toString(),
            }
        }
      });
    ```
````

File: docs/content/preview/quick-start/build-apps/nodejs/ysql-sequelize.md [hunk 4]
```diff
@@ -61,6 +62,24 @@ $ cd ./node/sequelize/
...
+    port: 5433,
+    host: "<host_name>",
+    dialectOptions: {
+        ssl: {
+            rejectUnauthorised: true,
+            ca: fs.readFileSync('<path_to_root_crt>').toString(),
+        }
+    }
+  });
+```
```
Comment (polarweasel) [inline docs/content/preview/quick-start/build-apps/nodejs/ysql-sequelize.md:82]: Markdownlint is helpful, and we have rules already defined for it in the repo...

(And again, should it be unauthoriZed?)

````suggestion

## Specifying SSL configuration

This configuration can be used while connecting to a YB Managed cluster or a local YB cluster with SSL enabled.

Use the configuration in the following way in the `models/index.js` file when you create the sequelize object:

```js
sequelize = new Sequelize("<db_name>", "<user_name>","<password>" , {
    dialect: 'postgres',
    port: 5433,
    host: "<host_name>",
    dialectOptions: {
        ssl: {
            rejectUnauthorised: true,
            ca: fs.readFileSync('<path_to_root_crt>').toString(),
        }
    }
  });
```
````

**Review by polarweasel — CHANGES_REQUESTED** [review]

**Comment by priyanshi-yb** [comment]
@aishwarya24, commit the suggested change.

File: docs/content/preview/drivers-orms/nodejs/sequelize.md [hunk 5]
```diff
@@ -191,6 +191,33 @@ Employees Details:
...
+    port: 5433,
+    host: "<host_name>",
+    dialectOptions: {
+        ssl: {
+            rejectUnauthorised: true,
+            ca: fs.readFileSync('<path_to_root_crt>').toString(),
+        }
+    }
+  });
+```
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/preview/drivers-orms/nodejs/sequelize.md]: @priyanshi-yb , Could you please confirm with Alex's question about the usage of rejectUnauthorised ? If it works , then please go ahead and merge this one.

**Pull request merged**